src/main/resources/application.properties

```diff
@@ -22,6 +22,8 @@ spring.kafka.properties.security.protocol=SSL
 spring.kafka.ssl.key-store-location=${KEYSTORE_LOCATION:file://timecard-restapi-keystore/timecard-restapi.keystore.jks}
 spring.kafka.ssl.key-store-password=${TIMECARD_KEYSTORE_PASSWORD}
 spring.kafka.ssl.key-password=${TIMECARD_KEYSTORE_PASSWORD}
+spring.kafka.ssl.trust-store-location=${TRUSTSTORE_LOCATION:}
+spring.kafka.ssl.trust-store-password=${TIMECARD_TRUSTSTORE_PASSWORD:}
```

Comment:
By adding the trust store to the main properties, breaks the main envs (dev, test , prod). The truststore is only needed for localdev.

Reply:
I have just tested this on your branch deploy please see main container logs.

Reply:
Well spotted @JWeston-HO ! I've addressed this, still without the need for a localdev profile. Please refer to the other PR (https://github.com/UKHomeOffice/callisto-localdev/pull/18) to see how this was implemented.
 I've tested the branch pod on callisto-dev. Feel free to retest.